demo/client/src/3d-demo.ts

```diff
@@ -406,21 +406,20 @@ class ThreeDView extends DemoView implements TextRenderContext {
               return;
 
           if (this.vrDisplay.isPresenting) {
-            const that = this;
             const eye = this.vrDisplay.getEyeParameters("left");
 
             this.vrDisplayHeight = eye.renderHeight;
             this.vrDisplayWidth = eye.renderWidth * 2;
             this.resizeToFit(true);
-            function vrCallback(): void {
-                if (that.vrDisplay == null || !that.renderer.inVR) {
+            const vrCallback = () => {
+                if (this.vrDisplay == null || !this.renderer.inVR) {
                     return;
                 }
-                that.vrDisplay.requestAnimationFrame(vrCallback);
-                that.inVRRAF = true;
-                that.redraw();
-                that.inVRRAF = false;
-            }
+                this.vrDisplay.requestAnimationFrame(vrCallback);
+                this.inVRRAF = true;
+                this.redraw();
+                this.inVRRAF = false;
+            };
```

Comment:
Ah, I see you made the arrow function change. Disregard the previous comments then :)